src/main/java/org/sagebionetworks/WorkflowManagerDocker.java

```diff
@@ -74,11 +65,11 @@ private ContainerRelativeFile createWorkflowParametersYamlFile(WorkflowParameter
 			File hostSynapseConfig) throws IOException {
 		File workflowParameters = createTempFile(".yaml", targetFolder.getContainerPath());
 		try (FileOutputStream fos = new FileOutputStream(workflowParameters)) {
-			IOUtils.write("submissionId: "+params.getSubmissionId()+"\n", fos, Charset.forName("UTF-8"));
-			IOUtils.write("workflowSynapseId: "+params.getSynapseWorkflowReference()+"\n", fos, Charset.forName("UTF-8"));
-			IOUtils.write("submitterUploadSynId: "+params.getSubmitterUploadSynId()+"\n", fos, Charset.forName("UTF-8"));
-			IOUtils.write("adminUploadSynId: "+params.getAdminUploadSynId()+"\n", fos, Charset.forName("UTF-8"));
-			IOUtils.write("synapseConfig:\n  class: File\n  path: "+hostSynapseConfig.getAbsolutePath()+"\n", fos, Charset.forName("UTF-8"));
+			IOUtils.write("submissionId: "+params.getSubmissionId()+"\n", fos, "UTF-8");
```

Comment:
See previous comment, probably there is some weird conflicting dependency. Use the StandardCharsets.UTF_8.